dev-docs/modules/qortexRtdProvider.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: page_v2
+title: Qortex Realtime Data Module
+display_name: Qortex RTD Module
+description: Appends contextual segments to the bidding object based on the content of a page
+page_type: module
+module_type: rtd
+module_code : qortexRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Qortex Real-time Data Submodule
+
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+```
+Module Name: Qortex RTD Provider
+Module Type: RTD Provider
+Maintainer: mannese@qortex.ai
+```
```

Comment:
All this meta information can be removed. It's part of the top level meta data.

```suggestion
```

You can add contact details at the bottom of the page